### ISSUE TITLE
Question about code

### ISSUE THREAD

**Issue by JKwon0331** [ml-explore/mlx-examples#618]
In line 120, mlx-examples/transofrmer_lm/main.py
I found the code that
optimizer.learning_rate = min(1, it / args.lr_warmup) * args.learning_rate

As I understand, this code is for the warmup.
However, after warmup, the learning rate is set to just args.learning_rate. Do you mean that you don't want to use weight_decay?

I am just wondering if it is okay to set the specific learning rate for my scheduler like this code, not implemented in mlx.
